config/examples/settings.yml

```diff
@@ -49,6 +49,7 @@ base: &default
 
 development:
   <<: *default
+  webpack_asset_host: http://localhost:3035
```

Comment:
No sure about this. If I don't set this to `<%= ENV.fetch('RAILS_ASSET_HOST', nil) %>` the packs are not taken from the CDN. I think we should either default `webpack_asset_host` to `RAILS_ASSET_HOST` or just remove `webpack_asset_host` and use the already existing `asset_host` declared a few lines above.

Reply:
OK, I'm not sure what it does so I'll remove it.

Reply:
How do we use "the already existing `asset_host`"

Reply:
@akostadinov We have `asset_host` already defined in settings.yml

https://github.com/3scale/porta/blob/4443a7840f6d6013c59da6dde1cc417907633b3b/config/examples/settings.yml#L5

What I meant is removing `webpack_asset_host` and wherever it was being used, use `asset_host` instead.

Reply:
m grep didn't show `webpack_asset_host` being used. Unless some library uses it.

Reply:
This PR added `webpack_asset_host` and all calls to it. If we are removing it, no calls to it should remain, so that's good.